packages/core/src/services/documentLogs/serialize.ts

```diff
@@ -1,32 +1,35 @@
-import { DocumentLog, SerializedDocumentLog } from '../../browser'
+import { DocumentLog, SerializedDocumentLog, Workspace } from '../../browser'
 import { database } from '../../client'
-import { findWorkspaceFromDocumentLog } from '../../data-access'
 import { NotFoundError, PromisedResult, Result } from '../../lib'
 import { ProviderLogsRepository } from '../../repositories'
 import { serialize as serializeProviderLog } from '../providerLogs'
 
 export async function serialize(
-  documentLog: DocumentLog,
+  {
+    workspace,
```

Comment:
We already had the `workspace` in the 2 places where this is used. Removing this responsibility to this service